lib/canvas_yaml.rb

```diff
@@ -28,6 +28,7 @@
 if defined?(YAML::ENGINE)
```

Comment:
My mistake. I see to_yaml is fixed below. Commit is updated with dead code removed.